File: test1_out/test1/ModFix/ModFix.c

/*
 * ======== string literals ========
 */

static const char test1_ModFix__s__fileName[] = "ModFix.p";
static const char test1_ModFix__s__0[] = "ModFix";

/*
 * ======== function definitions ========
 */

void test1_ModFix_foo__I() {
    uint8 len;
    bool f;
    uint32 i;
    len = 4;
    f = false;
    for ((i) = 0; (i) < (len); (i)++) {
        (f) = true;
    }
    for ((i) = 0; (i) < (len); (i)++) {
        (f) = true;
    }
}

void test1_ModFix_markUsed__E() {
}

void test1_ModFix_targetInit__I() {
}

File: test1_out/test1/ModFix/ModFix.js

/*
 * ======== MODULE test1.ModFix ========
 */

var $$c = function() {
    const ModFix = {};
    ModFix.ModFix = ModFix
    ModFix.$name = 'test1.ModFix';
    ModFix.pollen$used = 0;
    ModFix.foo = new $$Ref('test1_ModFix_foo__I');
    ModFix.markUsed = new $$Ref('test1_ModFix_markUsed__E');
    ModFix.targetInit = new $$Ref('test1_ModFix_targetInit__I');
    ModFix.$$hostInit = function() {
        var $$text = '';
        return $$text;
    }
    ModFix.pollen__uses__ = function() {
    }
    ModFix.$$privateInit = function() {
    }
    return ModFix;
}
var $$u = $$c();
$units['test1.ModFix'] = $$u;
$units.push($$u);

